astrodb_utils/fits.py

```diff
@@ -299,14 +315,18 @@ def make_skycoord(header):
             print("DEC_TARG or DEC is required")
```

Comment:
This can be done more cleanly by checking those against `header.keys`, something like `'DEC' in header.keys() or 'DEC_TARG' in header.keys()` instead of trying to store them in different variables and having a lot of IF statements.

Reply:
Thanks -- working on this! I haven't figured it out yet tho. Cause it's also not just that the keyword is present, but that's it's not None.